src/styles/themer/index.js

```diff
@@ -16,6 +21,7 @@ class Themer {
 
   set themeConfig(themeConfig) {
```

Comment:
Nit and getting mostly into personal preference, but especially given that `themer.themeConfig = { ... }` now triggers a callback as well, I'm less of a fan of the `set themeConfig` api. Triggering a callback on an assignment might be confusing. Don't necessarily feel strongly enough to change it though

Reply:
I think it works pretty well. The other way this could work would an older style set
```js
themer.setConfig(config)
```

I personally like the setter because this way there's no way to muck with the internal theme even if you really want to.